src/lib/gist_backup.sh.in

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# gist_backup.sh : Enables backing up package list on github gist
+# This file is part of Yaourt (http://archlinux.fr/yaourt-en)
```

Comment:
My 2 cents: `yaourt` source files are indented using a tab character. You should keep this convention for this file too.